src/lib/Libwin/passwd.c

```diff
@@ -757,7 +777,11 @@ get_dcinfo(char *net_name,
 		if (dctrl) {
 			strncpy(domain_name, dctrl->DomainName, PBS_MAXHOSTNAME);
 			strncpy(domain_ctrl, dctrl->DomainControllerName, PBS_MAXHOSTNAME);
-			NetApiBufferFree(dctrl);
+			nBufferStat = NetApiBufferFree(dctrl);
```

Comment:
for this NetApiBufferFree() I suggest creating wrapper as in https://github.com/PBSPro/pbspro/pull/1727
Since there are many NetApiBufferFree() invocations in this file, with wrapper it will look clean

Reply:
Really like the idea, it would help much.
But as per latest team discussion, it probably not a good idea to add code no and this api is mainly used only in passwd.c (other Libwin files don't use this).
Probably we can think of this is future refactoring